Assets/Scripts/PauseManagerScript.cs

```diff
@@ -43,13 +48,18 @@ public void Pause(int joystick)
             // Reset menu
             es.SetSelectedGameObject(null);
             es.SetSelectedGameObject(es.firstSelectedGameObject);
+
+            // SFX
             MusicManagerScript.Instance.TurnOffEverything();
-            SoundManagerScript.instance.muteSFX();
```

Comment:
Does removing this line cause currently playing SFX (like the anti-grav warning siren) to continue to play through the pause menu?

Reply:
Hmm, it doesn't seem to have this effect, but I can't figure out why!

Reply:
You know I thought it might have this effect but I couldn't think of another way to allow SFX with the menu open. Weird that it's not happening!